firmware/common/include/interface/blitter.h

```diff
@@ -14,14 +14,30 @@
 #define _BLITTER_H
 
 struct BlitterArea {
-	uint16_t 	address;
-	uint8_t 	page,_filler;
-	uint32_t  	stepSize;
-	uint32_t  	stepOffset;
-	uint32_t  	stepCount;
+	uint16_t	address;
+	uint8_t		page;
+	uint8_t		format;     // one of BLTFMT_*
+	uint8_t		k;			// mask in src, solid value in target.
+	uint8_t  	width;		// Size of line, in units (ignored on target).
```

Comment:
Using byte for the width, limits area ro 255 bytes of width. and height is word, where our vertical screen width is 240. I suggest to swap data types for those two values.